### PR TITLE
(#133) - Add Russian language support

### LINKED ISSUE #133: Add Russian language support
Translate text into the Russian.

### PR DESCRIPTION
**Pull request by wazsone**
Closes: #133

### CODE REVIEW

File: app/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    <string-array name="snooze_options">
+        <item>10 Минут</item>
+        <item>30 Минут</item>
+        <item>1 Час</item>
+    </string-array>
+
+    <string name="snooze">Отложить</string>
+    <string name="remove">Удалить</string>
+    <string name="todo_error">Пожадуйста введите Напоминание</string>
```
Comment (gendolf3d) [inline app/src/main/res/values-ru/strings.xml:31]: Please change to:
Пожалуйста, введите напоминание

File: app/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <string name="choose_date">Дата</string>
+    <string name="choose_time">Время</string>
+    <string name="remind_date_and_time">Напоминание установлено на %1$s, %2$s %3$s</string>
+    <string name="about">О приложении</string>
+    <string name="app_version">Версия: %1$s</string>
+    <string name="contact_me">Вы можете связаться с мной по</string>
+    <string name="made_by">Создатель Avjinder</string>
+    <string name="done_label">Готово</string>
+    <string name="date_reminder_default">Сегодня</string>
+    <string name="date_error_check_again">Вы ввели некорректную дату.</string>
```
Comment (gendolf3d) [inline app/src/main/res/values-ru/strings.xml:17]: Please remove full stop `.`

File: app/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <string name="action_settings">Настройки</string>
+    <string name="no_to_dos">У вас нет напоминаний</string>
+    <string name="title">Заголовок</string>
+    <string name="remind_me">Напомнить мне</string>
+    <string name="choose_date">Дата</string>
+    <string name="choose_time">Время</string>
+    <string name="remind_date_and_time">Напоминание установлено на %1$s, %2$s %3$s</string>
+    <string name="about">О приложении</string>
+    <string name="app_version">Версия: %1$s</string>
+    <string name="contact_me">Вы можете связаться с мной по</string>
```
Comment (gendolf3d) [inline app/src/main/res/values-ru/strings.xml:13]: Please fix typo in: `с мной`

File: app/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (wazsone) [reply app/src/main/res/values-ru/strings.xml:31]: Fixed in https://github.com/avjinder/Minimal-Todo/pull/139

File: app/src/main/res/values-ru/strings.xml (hunk 2 above)
Comment (wazsone) [reply app/src/main/res/values-ru/strings.xml:17]: Fixed in https://github.com/avjinder/Minimal-Todo/pull/139

File: app/src/main/res/values-ru/strings.xml (hunk 3 above)
Comment (wazsone) [reply app/src/main/res/values-ru/strings.xml:13]: Fixed in https://github.com/avjinder/Minimal-Todo/pull/139

**Comment by wazsone** [comment]
Fixed in https://github.com/avjinder/Minimal-Todo/pull/139

Wasn't able to restore previous fork and branch. So had to do it again.

I'm sorry for inconvenience.

**Pull request closed without merging**